### DISCUSSION TITLE
Strange behavior from field light on Gizmo

### DISCUSSION THREAD

**Discussion by jds-designs** [gizmo-platform/gizmo#22] [Q&A] [answered]
Hello, I have been having trouble pairing my hub's first Gizmo board to a driver station. After following the process described in [Section 3.4 of the docuemtation](https://gizmoplatform.dev/startup/firmware.html), the Network light (C3) is yellow, and the Field light (C2) is on and fading repeatedly from red, to green, to blue, back to red, and so on. It did not flash white like I expected. The Battery light (C1) is green. I've included a video to show this. The lights stayed in this state for over 30 minutes.

This behavior is the same whether the driver station is connected wirelessly, or connected via ethernet cable.

I tried to use the BEST default program to control some motors via the joystick, but they didn't respond.

I have double checked to make sure I did all the configuration and firmware installation in [Section 3 of the documentation](https://gizmoplatform.dev/startup/index.html) correctly. I went through all the steps a second time to make sure, referencing the video guides in the [BESTedu getting started course](https://bestedu.bestrobotics.org/courses/gizmo-setup-for-hubs/lessons/).

I was unable to find any documentation of what the fading colors on the Field light means. Is there a setup mistake that can lead to this?

Thank you!

https://github.com/user-attachments/assets/c243f505-a8be-4774-8710-07b175134fe9

**Comment by barulicm** [comment]
When you are seeing this behavior, is the gamepad connected to your driver station?

The yellow network light indicates that the Gizmo is connected to the driver station's network but is not receiving data from the driver station's control software. The field light shows the fading animation under this condition as well.

Last week's release for the driver station and Gizmo changed some behavior when problems with the gamepad emerge, and this light pattern could be consistent with a driver station that does not have its gamepad plugged in.

**Comment by jds-designs** [comment]
Yes, the gamepad is connected when I'm seeing this behavior.

I'm using [firmware v0.1.0 for the system processor](https://github.com/gizmo-platform/firmware/releases/tag/v0.1.0), and [firmware v0.1.2-rc1 for the driver station](https://github.com/gizmo-platform/gizmo/releases/tag/v0.1.2-rc1).

**Comment by barulicm** [comment, marked as answer]
From off-platform discussion, the problem ended up being the switch on the back of the gamepad. It needs to be in the "D" position.
